fire/cli/info.py

```diff
@@ -203,7 +204,7 @@ def observation_linje(obs) -> str:
     fra = kanonisk_ident(obs.opstillingspunktid)
     til = kanonisk_ident(obs.sigtepunktid)
     dH = obs.value1
-    L = obs.value2
+    L = obs.value2 if obs.value2 > 0.001 else 0.001
```

Comment:
```suggestion
    L = max(obs.value2, 0.001)
```
Det er vist et ækvivalent udtryk. Efter min mening mere læsbart, da du undgår Pythons uskønne ternary operator (som er grunden til at jeg ikke er 100% sikker på at mit forslag gør det samme som koden her).

Reply:
Klart mere læsbart!